src/api/app/views/webui/users/tasks/index.html.haml

```diff
@@ -1,85 +1,78 @@
-- @pagetitle = "Tasks for #{User.session!}"
-- @layouttype = 'custom'
-- @crumb_list = [@pagetitle]
+:ruby
+  @pagetitle = "Tasks for #{User.session!}"
+  @crumb_list = [@pagetitle]
```

Comment:
`@crumb_list` is only for the old UI, so this should be needed

Reply:
removed everything since in this page we don't need `@pagetitle` too

Reply:
`@pagetitle` is [displayed in the layout](https://github.com/openSUSE/open-build-service/blob/e9a4d1f8cb8cfb047572d425ab2cd57660548630/src/api/app/views/layouts/webui/webui.html.haml#L10), so we should definitely keep it.

Reply:
it is displayed *if* there. I don't think that I need to repeat the Breadcrumb there. What do you think @dmarcoux ?

![Screenshot_2019-09-16_11-06-15](https://user-images.githubusercontent.com/37418/64946329-ea487800-d872-11e9-995a-2a0f3920d154.png)